src/styles.scss

```diff
@@ -16,6 +16,21 @@ html, body {
     font-style: normal;
 }
 
+.content{
+    max-width: 1500px;
```

Comment:
Flowchart needs to have a max-width as well. At the moment it is going out of bounds